pyiron_contrib/atomistic/atomistics/job/simple_atomistic.py

```diff
@@ -0,0 +1,66 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from pyiron_atomistics.atomistics.job.interactive import GenericInteractive
+import numpy as np
+
+class SimpleAtomistic(GenericInteractive):  # Create a custom job class
+    def __init__(self, project, job_name):
+        super().__init__(project, job_name)
+        self.neigh = None
+        self.potential = None
+        self._neighbor_chemical_symbols = None
+        self.uptodate = False
+
+    def interactive_initialize_interface(self):
+        pass
+
+    def validate_ready_for_run(self):
+        super(SimpleAtomistic, self).validate_ready_for_run()
```

Comment:
nit: moderns `super` calls don't need the arguments (unless you have dual inheritance and want non-default resolution order)